lib/lexer_mdx.mll

```diff
@@ -69,6 +69,19 @@ rule text section = parse
            List.iter (fun _ -> newline lexbuf) errors;
            newline lexbuf);
         `Block block :: text section lexbuf }
+  | "<!--" ws* "$MDX" ws* ([^' ' '\n']* as label_cmt) ws* "-->" ws* eol
```

Comment:
It would be great if this regex could be shared between the two rules. I'm not sure if it's possible to do with ocamllex, we should probably split lexing and parsing.

Reply:
I had a look at this but didn't find a simple way to do it, the long term goal would be to switch to `omd` to parse all that anyway